Core.ApplicationServices/Model/Contracts/Write/ItContractGeneralDataModificationParameters.cs

```diff
@@ -15,5 +15,7 @@ public class ItContractGeneralDataModificationParameters
         public OptionalValueChange<Maybe<bool>> EnforceValid { get; set; } = OptionalValueChange<Maybe<bool>>.None;
         public OptionalValueChange<Maybe<DateTime>> ValidFrom { get; set; } = OptionalValueChange<Maybe<DateTime>>.None;
         public OptionalValueChange<Maybe<DateTime>> ValidTo { get; set; } = OptionalValueChange<Maybe<DateTime>>.None;
+        public OptionalValueChange<string> CriticalityTypeId { get; set; } = OptionalValueChange<string>.None;
+        public OptionalValueChange<Guid?> CriticalityTypeUuid { get; set; } = OptionalValueChange<Guid?>.None;
```

Comment:
none of these two properties are used in the write service, so how can this be ready for review?